src/components/SettingsModal.js

```diff
@@ -12,6 +20,36 @@ export default function SettingsModal({ setDisplayModal }) {
           className="modal-close-button"
         />
         <h1 className="modal-header">Settings</h1>
+
+        <div className="modal-range-container">
+          <label htmlFor="delay-slider">
+            Set how frequently the app will fetch new places. Values are in
+            seconds.
+          </label>
+          <div className="modal-range-pic-container">
+            <span className="modal-range-pic">
+              <img src="./img/slider-car.png" alt="car-emoji" />
+            </span>
+            <span className="modal-range-pic">
+              <img src="./img/slider-bike.png" alt="bike emoji" />
+            </span>
+            <span className="modal-range-pic slider-walker">
+              <img src="./img/slider-walker.png" alt="walker emoji" />
+            </span>
+          </div>
+          <input
+            type="range"
+            id="delay-slider"
+            className="delay-slider"
+            name="delay-slider"
+            min="0"
+            max="30"
+            value={sliderValue}
+            step="5"
+            onChange={(e) => handleSliderChange(e)}
+          />
+          <div className="slider-value">{sliderValue}</div>
```

Comment:
This feels like it needs an ARIA role or label so screen readers can find it.

Reply:
I'm still learning about the best way to use ARIA, so hopefully the update is an improvement, but open to further feedback or suggestions.